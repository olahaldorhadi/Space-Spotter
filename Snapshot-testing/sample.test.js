test('should match snapshot', () => 

{
const data = {name: 'John',age: 30,email: 'john@example.com'};
expect(data).toMatchSnapshot();
});

