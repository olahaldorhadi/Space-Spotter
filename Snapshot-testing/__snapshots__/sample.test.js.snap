// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
{
  "age": 30,
  "email": "john@example.com",
  "name": "John",
}
`;
